controllers/network/ipset_controller.go

```diff
@@ -432,8 +433,8 @@ func (r *IPSetReconciler) ensureReservation(
 		}
 
 		if ipsetNet.FixedIP != nil {
-			ipDetails.FixedIP = net.ParseIP(string(*ipsetNet.FixedIP))
-			if ipDetails.FixedIP == nil {
+			ipDetails.FixedIP = netip.MustParseAddr(string(*ipsetNet.FixedIP))
```

Comment:
`MustParseAddr` panics on error, is this intended to be used? thats why we used the ParseIP version which does not panic

Reply:
Will switch to ParseAddr... MustParseAddr should only be used for tests :)